pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: reflectable
-version: 2.2.1+1
+version: 2.2.1+2
```

Comment:
For future similar changes this should be a patch version bump (`2.2.2`) instead of appending a `+1` or `+2`.

See https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance#bumping-versions